code/__DEFINES/dcs/signals.dm

```diff
@@ -263,6 +265,13 @@
 #define COMSIG_WARRIOR_CTRL_CLICK_ATOM "warrior_ctrl_click_atom"
 	#define COMSIG_WARRIOR_USED_LUNGE 1
 
+#define COMSIG_XENOMORPH_PLASMA_REGEN "xenomorph_plasma_regen"
+
+#define COMSIG_XENOMORPH_ZONE_SELECT "xenomorph_zone_select"
+	#define COMSIG_ACCURATE_ZONE 1
```

Comment:
```suggestion
	#define COMSIG_ACCURATE_ZONE (1<<0)
```

Reply:
please make a pr to change all the signal returns to bitshifts if this matters to you

Reply:
Cobby did this on /tg/ actually for this file.
`src.stuff` and not using numbers for booleans is consistent with old code as well, yet we improve it when we touch upon things or introduce new stuff.

Might need to add it to the contribution guidelines to use bitflag notation for bitflags same as the boolean defines for bools.

Reply:
pr it here please too.

Reply:
Will do, but try to keep new additions like this one following the format as well.

As you can see right now it's a mix of both styles, there's both `#define COMPONENT_NO_STUN (1<<0)` and `#define COMSIG_WARRIOR_USED_LUNGE 1`

Also, use `COMPONENT_NAME` for returns, not `COMSIG_NAME`, that's for the string names.

Reply:
uh
```
#define COMSIG_MOB_CLICK_ALT "mob_click_alt"					//from base of mob/AltClickOn(): (atom/A)
	#define COMSIG_MOB_CLICK_CANCELED (1<<0)
	#define COMSIG_MOB_CLICK_HANDLED (1<<1)
```

Reply:
uh what? If you keep checking you'll most of them use the `COMPONENT_` pattern, there's like only 3 examples of the `COMISG_` one on /tg/ only?
This is a pretty simple change request to use a single standard.

Reply:
sure, document it cause the thing i linked was commited by you a few months ago

Reply:
My bad.